x-pack/plugins/features/server/plugin.test.ts

```diff
@@ -23,15 +23,15 @@ coreStart.savedObjects.getTypeRegistry.mockReturnValue(typeRegistry);
 describe('Features Plugin', () => {
   it('returns OSS + registered features', async () => {
     const plugin = new Plugin(initContext);
-    const { registerFeature } = await plugin.setup(coreSetup, {});
+    const { registerKibanaFeature: registerFeature } = await plugin.setup(coreSetup, {});
```

Comment:
Could we add a simple test for the added `registerElasticsearchFeature` + `getElasticsearchFeatures` APIs?

NIT: use `registerKibanaFeature` in test code instead of spread-rename to `registerFeature`